fixes issues with current Issue Templates that were created in haste (#14)
sandialabs/Gulliver: .github/ISSUE_TEMPLATE/feature_request.md

---

# Description

A clear and concise description of what the desired feature is. 

> Is your feature request related to an exiting  problem? Please describe it and/or provide an issue number.

Reviewer: ```suggestion
Is your feature request related to an existing problem? Please describe it and/or provide an issue number.
```

# Proposed solutions

Do you have a solution, what would you recommend to fix it?
If you don't have a solution what would you like the end result to be?

# Can you help?

> Let us know if you are interested in helping, and in what capacity. eg: I want to write code and create a pull request, I can provide test cases, I have a suggestion on how it can be done, etc

Reviewer: ```suggestion
Let us know if you are interested in helping, and in what capacity. e.g.: I want to write code and create a pull request, I can provide test cases, I have a suggestion on how it can be done, etc.
```

# Describe alternatives you've considered

Did you consider any alternatives solutions or features?
Did you have a workaround that met your needs?

> 

Reviewer: ```suggestion
```
